.github/workflows/ci.yml

```diff
@@ -564,6 +562,7 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: rm -rf ~/.m2/repository
 
+
```

Comment:
unrelated, please revert (this is in `Enable Maven cache in build-test-matrix job of ci.yml` commit)